tests/providers/google/cloud/operators/test_dataproc.py

```diff
@@ -763,6 +854,23 @@ class TestDataProcSparkOperator(unittest.TestCase):
         "spark_job": {"jar_file_uris": jars, "main_class": main_class},
     }
 
+    def setUp(self):
+        self.dagbag = DagBag(
+            dag_folder='/dev/null', include_examples=False
+        )
+        self.dag = DAG(TEST_DAG_ID, default_args={
+            'owner': 'airflow',
+            'start_date': DEFAULT_DATE
+        })
+        self.mock_context = MagicMock()
+
+    def tearDown(self):
+        session = Session()
+        session.query(TaskInstance).filter_by(
+            dag_id=TEST_DAG_ID).delete()
+        session.commit()
+        session.close()
```

Comment:
Same here, we can use existing methods from `tests.test_utils.db`. And here we also should clear XCom to avoid side effect that is created on L953

Reply:
Done using test_utils